crypto/s2n_rsa.c

```diff
@@ -124,15 +90,15 @@ static int s2n_rsa_modulus_check(RSA *rsa)
     return 0;
 }
 
-int s2n_rsa_public_encrypted_size(struct s2n_rsa_public_key *key)
+int s2n_rsa_public_encrypted_size(const s2n_rsa_public_key *key)
 {
     notnull_check(key->rsa);
     GUARD(s2n_rsa_modulus_check(key->rsa));
 
     return RSA_size(key->rsa);
 }
 
-int s2n_rsa_private_encrypted_size(struct s2n_rsa_private_key *key)
+int s2n_rsa_private_encrypted_size(const s2n_rsa_private_key *key)
```

Comment:
These were originally written as distinct functions probably to make it more readable for the callers. I think they also reflect conceptually what is happening (encryption with the public key vs encryption with the private key). I've made a common static function that both use to mitigate your concern about code duplication.